### DISCUSSION TITLE
is this project related to https://github.com/webview/webview at all?

### DISCUSSION THREAD

**Discussion by laoshaw** [tauri-apps/wry#524] [General]
https://github.com/webview/webview

**Comment by nothingismagick** [comment]
A long time ago in the early phases of the tauri project, we started working on helping the webview/webview project get back on its feet. We discovered many limitations and the complexities of a variety of languages that were just not as safe to reason about as it would have been to rewrite it in rust. So that is what we did.

**Comment by laoshaw** [comment]
is it a good idea to use wry to replace webview/webview? is wry specifically made for Tauri or is it general enough for what webview/webview does(e.g. providing golang/c/c++/python bindings)

**Reply by nothingismagick** [reply]
wry is currently rust only, however its stability and completeness is a few steps ahead of what webview/webview offers. that said, the work on creating bindings has not yet been undertaken, but that is more due to a lack of contributions than it is due to the challenge of making said bindings.

**Reply by nothingismagick** [reply]
On a sidenote, please see the work here at using tauri as a dynamic library:

https://github.com/tauri-apps/tauri/tree/dev/examples/tauri-dynamic-lib
